releases/v2.0-beta.20180319.md

```diff
@@ -0,0 +1,108 @@
+---
+title: What&#39;s New in v2.0-beta.20180319
+toc: false
+summary: Additions and changes in CockroachDB version v2.0-beta.20180319 since version v2.0-beta.20180312
+---
+
+## March 19, 2018
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.0-beta.20180305-237-ge0b123c23.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
```

Comment:
@lhirata, you need to update the dates in the binary links: `v2.0-beta.20180319.darwin...`, etc.